docs/discover/context.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[discover-document-context]]
+== View surrounding documents
+
+Once you've narrowed your search to a specific event in *Discover*,
+you can inspect the documents that occurred
+immediately before and after the event.
+To view the surrounding documents, your index pattern must contain time-based events.
+
+. In the document table, click the expand icon (>).
+. Click *View surrounding documents.*
++
+In the context view, documents are sorted by the time field specified in the index pattern
+and displayed using the same set of columns as the *Discover* view from which
+the context was opened. The anchor document is highlighted in blue.
++
+[role="screenshot"]
+image::images/Discover-ContextView.png[Image showing context view feature, with anchor documents highlighted in blue]
++
+The filters you applied in *Discover* are carried over to the context view.
++
+[role="screenshot"]
```

Comment:
My thinking is that we only need the second image of the filters because it shows both an enabled and a disabled filter.

Reply:
👍 done